browser-test/src/applicant/applicant_prevent_duplicate_submission.test.ts

```diff
@@ -102,7 +102,6 @@ test.describe('Prevent Duplicate Submission', () => {
 
       await test.step('Submit another application to the same program without changing anything', async () => {
         await applicantQuestions.clickApplyProgramButton(programName)
-        await applicantQuestions.clickContinue()
```

Comment:
**question (non-blocking):** Are you removing these `clickContinue()` because it's an unnecessary step?

Reply:
Ah, now I get it.  Thanks for explaining.